uai2017/python/metadata-papers.py

```diff
@@ -53,11 +54,13 @@ def prettify(elem):
 bids_by_number = defaultdict(list)
 bids_by_id = defaultdict(list)
 
+print "Processing bids... (this may take a while)"
 for b in bids:
     n = client.get_note(b.forum)
```

Comment:
why don't you get the notes from the submissions call? instead of doing call per bid